src/main/scala/scorex/core/network/peer/PenaltyType.scala

```diff
@@ -0,0 +1,27 @@
+package scorex.core.network.peer
+
+sealed trait PenaltyType {
```

Comment:
Please add class and it's fields description

Reply:
Done